bot/exts/evergreen/pythonfacts.py

```diff
@@ -0,0 +1,28 @@
+import random
+
+import discord
+from discord.ext import commands
+
+
+with open('bot/resources/evergreen/python_facts.txt') as file:
+    FACTS = list(file)
+
+COLORS = [0x4B8BBE, 0xFFD43B, ]
+
+
+class PythonFacts(commands.Cog):
+    """Sends a random fun fact about Python."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @commands.command(name='pythonfact', aliases=['pyfact'])
+    async def get_python_fact(self, ctx: commands.Context) -> None:
+        """Sends a Random fun fact about Python."""
+        embed = discord.Embed(title='Python Facts', description=random.choice(FACTS), colour=random.choice(COLORS))
```

Comment:
I meant using `itertools.cycle`, and break this embed into multiple lines for better readability.

Reply:
What’s that?

Reply:
It basically cycles through the element of the list one by one, like here is a example
```python
>>> import itertools
>>> shape_list = ["square", "triangle", "circle", "pentagon", "star", "octagon"]
>>> g = itertools.cycle(shape_list)
# Then call next() whenever you want another one.
>>> next(g)
'square'
>>> next(g)
'triangle'
>>> next(g)
'circle'
>>> next(g)
'pentagon'
>>> next(g)
'star'
>>> next(g)
'octagon'
>>> next(g)
'square'
>>> next(g)
'triangle'
```